src/ngrest/render/RenderCrud.php

```diff
@@ -32,6 +32,26 @@ class RenderCrud extends Render implements ViewContextInterface, RenderCrudInter
 
     const TYPE_UPDATE = 'update';
 
+    /**
+     * @var string The view file for create and update form rendering. Aliases are supported.
+     * By default the location for this file is the viewPath {@see getViewPath()}
+     *
+     * @since 2.0
+     */
+    public $crudFormView = '_crudform';
```

Comment:
@denyadzi i think it is good that someone can change this view, but we should also add a strong and bold information that this is not recommend.

Why it's not recommend?

One of the strongest arguments for the LUYA admin is that you can run composer update and you get fresh updates, improvments UI fixes, new buttons, etc. - if you override the view files a next update can

+ break the application, as we don't take care of BC breaks in view files (as we don't recommend people to override views)
+ won't profit from further updates.

So yes, its okay to make it possible, but we should also warn people not to do so ;-)

@luyadev/maintainer suggestions?

Reply:
(today for example, i made a change which would break all the overriden templates if you are not going to track all changes by your self inside those views: https://github.com/luyadev/luya-module-admin/commit/d69e2f5a67a1a9ce6e379b2deb90536e2553ff1c#diff-cba1795d3559c80b399b58899289426b <-- _crudform.php)